modules/data.atmosphere/tests/testthat/data/test.met2CF.csv.csv

```diff
@@ -1,24 +1,24 @@
 Corrected Date/Time,Solar Radiation (W/m2),Temp (C),PAR_(umol_m-2_s-1),PAR_(mol_m-2_h-1),RH (%),Wind Speed (m/s),VPD,Rain (mm)
-3/26/13 18:00,262.5,4.004,553.075,1.99107,46.175,2.645,0.437784909,0
-3/26/13 19:00,91.25,3.06175,183.075,0.65907,50.125,2.265,0.379530354,0
-3/26/13 20:00,2.8,1.24975,6.2,0.02232,55.475,0.315,0.297662391,0
-3/26/13 21:00,0.6,1.08475,0,0,55.475,0.5675,0.294143987,0
-3/26/13 22:00,0.6,0.701,0,0,60.975,0.44,0.250761625,0
-3/26/13 23:00,0.6,0.76975,0,0,62.6,0.82,0.241517855,0
-3/27/13 0:00,0.6,-0.92425,0,0,70.5,0.0625,0.168393094,0
-3/27/13 1:00,0.6,-0.65475,0,0,73.075,0,0.156759054,0
-3/27/13 2:00,0.6,0.48025,0,0,77.025,0.44,0.145288538,0
-3/27/13 3:00,0.6,-0.03225,0,0,84.6,0.0625,0.093827301,0
-3/27/13 4:00,0.6,0.18275,0,0,77.625,0.3125,0.13847215,0
-3/27/13 5:00,0.6,-0.103,0,0,75.15,0.25,0.150625162,0
-3/27/13 6:00,0.6,-0.77475,0,0,78.25,0.125,0.125522207,0
-3/27/13 7:00,1.55,-0.64075,4.325,0.01557,77.4,0.3125,0.131713492,0
-3/27/13 8:00,89.675,-0.48025,161.825,0.58257,76.975,0.3775,0.135774498,0
-3/27/13 9:00,183.425,0.54275,382.45,1.37682,73.1,1.07,0.170881471,0
-3/27/13 10:00,280,0.97525,590.575,2.12607,72.075,1.885,0.183028309,0
-3/27/13 11:00,405.025,2.2,860.575,3.09807,69.425,1.51,0.218820454,0
-3/27/13 12:00,471.25,3.1465,1021.825,3.67857,65.65,1.1975,0.26296636,0
-3/27/13 13:00,391.55,4.88,868.075,3.12507,59.8,1.76,0.347681221,0
-3/27/13 14:00,319.7,3.17725,684.95,2.46582,70.175,2.14,0.228823375,0
-3/27/13 15:00,138.75,1.17875,325.575,1.17207,88.55,0.3775,0.076156034,0.4
-3/27/13 16:00,135.625,1.85725,310.575,1.11807,87.075,0.8175,0.090261398,0
+2013-03-26 18:00,262.5,4.004,553.075,1.99107,46.175,2.645,0.437784909,0
```

Comment:
I was not suggesting to change the format of the test file, just in `start_date` and `end_date`. I don't think having the file be in ISO 8601 would hurt anything, but I liked how the existing setup gave a nice confirmation that (1) met2CF.csv can convert nonstandard date formats if they're correctly specified at call time, and (2) the start and end need not be specified in the same format stored in the file -- none of these will be compared to each other until after they're parsed to R's internal datetime format.

Reply:
Fixed. Can you check it now. All existing and present tests pass